docs/cookbook/languages/python.md

```diff
@@ -308,7 +309,48 @@ current directory.
 If you're working in a repository with multiple packages in subdirectories,
 you would replace `.` with the path to the package sources.
 
-This works for projects using `pyproject.toml` as well (including Poetry) as
-long as the `[build-system]` section of `pyproject.toml` is filled out.
+### Poetry
```

Comment:
unrelated to your PR, but is pip really that easy? I guess we should add that to manifest-build-examples